content/en/docs/collector/configuration.md

```diff
@@ -39,6 +39,16 @@ otelcol --config=env:MY_CONFIG_IN_AN_ENVVAR --config=https://server/config.yaml
 otelcol --config="yaml:exporters::debug::verbosity: normal"
 ```
 
+You can also provide multiple configurations using multiple files at different
+paths. Each file can be a full/partial configuration and reference components
+from each other. For simplicity, if the merge of files does not constitute a
+complete configuration user will get an error since the defaults are not added
+by default. For example:
+
+```shell
+otelcol --cofig=file:/path/to/first/file --config=file:/path/to/second/file
+```
```

Comment:
I've made a few edits to this paragraph, but I also think the whole section needs to be moved a little higher on the page where it will flow better with the other examples. Could you shift this paragraph and its example so that they come after line 32? Thanks!

```suggestion
You can also provide multiple configurations using multiple files at different
paths. Each file can be a full or partial configuration, and the files can reference components
from each other. If the merger of files does not constitute a
complete configuration, the user receives an error since required components are not added
by default. Pass in multiple file paths at the command line as follows:

```shell
otelcol --config=file:/path/to/first/file --config=file:/path/to/second/file
```
```